packages/connect-core/src/utils/descriptions.ts

```diff
@@ -1,76 +1,76 @@
-import { ethers } from 'ethers'
```

Comment:
To prevent errors with types I decided to comment the logic related to the Forwarding Path descriptions.